### PR TITLE
America flag, America Rhino And Enclave LT New Loadout

### PR DESCRIPTION
**Pull request by projectkepler-ru**
## About The Pull Request
Adding America Flag proper alongside chiappa rhino (currently only limited to the enclave LT new loadout) oh and placing it on the map

## Why It's Good For The Game
I wanted a new enclave LT loadout so I'll do it myself
We could use America Flag
And Chiappa Rhino is my favourite gun so I figured I'd also add it myself 
This should not at all impact balances

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added the LT 'Golden Hand' Loadout, This contains a golden sniper rifle that only has half of the slowdown instead of none at all, this make it still slower than the venator and the actual golden sniper rifle, but slightly faster than the compact sniper rifle.  The chiappa Rhino deals 55 damage and is chambered in .44 magnum. it fires slightly faster than the magnum revolver.
imageadd: Sprite for the Chiappa Rhino and America Flag
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 1]
```diff
@@ -1047,6 +1047,17 @@
 	fire_delay = 2.60
 	can_bayonet = TRUE
 
+/obj/item/gun/ballistic/automatic/marksman/sniper/america
+	name = "Gobi Campaign sniper rifle"
+	desc = "A Customized DKS 501, chambered in .308 Winchester. coated in gold paint this one was made for the Gobi Campaign, There's a torn rag from a flag wrapped around it's stock"
```
Comment (ProbablyCarl) [inline code/modules/projectiles/guns/ballistic/automatic.dm:1052]: "it's" to "its", as it's a possessive.

File: code/modules/projectiles/guns/ballistic/revolver.dm [hunk 2]
```diff
@@ -226,6 +226,15 @@
 	icon_state = "mysterious_m29"
 	can_scope = FALSE
 
+//Chiappa Rhino                  Keywords: Enclave, .44, Double action, 6 rounds cylinder
+/obj/item/gun/ballistic/revolver/m29/rhino_america
+	name = "Rhino America"
+	desc = "A revolver machined from a solid aluminium block, This one is coated in gold paint with it's grip painted to look like the flag of America. This model has been specifically made to fire .44 magnum cartridges"
```
Comment (ProbablyCarl) [inline code/modules/projectiles/guns/ballistic/revolver.dm:232]: As above.

**Comment by projectkepler-ru** [comment]
aaaaa hold on

**Comment by projectkepler-ru** [comment]
@ProbablyCarl

**Pull request merged**